python/document_reader_fine_tuning.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
+"""Finetuning.ipynb
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/1PRCbaNy2B4vs_3oO2PaOSswJPZUL0PTD
+"""
+
+# pip install datasets
+# pip install transformers
+# pip install huggingface_hub
+# pip install evaluate
+# pip install torch
+
+import os
+import enum
+import ast
+from utils import HUGGINGFACE_TOKEN, DocReader, Sports
+
+from datasets import load_dataset, Dataset, DatasetDict
+from transformers import (
+    AutoTokenizer,
+    AutoModelForQuestionAnswering,
+    DefaultDataCollator,
+    TrainingArguments,
+    Trainer,
+    EarlyStoppingCallback,
+    pipeline,
+)
+from huggingface_hub import HfApi, login
+
+DOC_READER = DocReader.DistilBERT
+SPORT = Sports.BASKETBALL
+
```

Comment:
Please, add a comment explaining the propose of these variables.
After the selection, add a `print(SPORT.name, DOC_READER.name)`